style.css

```diff
@@ -482,6 +592,116 @@ textarea {
     width: 170px;
   }
 
+  /* start of pop up  */
+  .poped-section {
+    width: 100%;
+    height: 100vh;
+    overflow-y: scroll;
+    background-color: #091e423a;
+  }
+
+  .ctn {
+    background-color: #fff;
+  }
+
+  .active-menu {
+    display: block;
+  }
+
+  .project-container {
+    position: relative;
+    height: 100%;
+    width: 100%;
+    border-radius: 0;
+    background-color: #091e423a;
+  }
+
+  .pops-project {
+    width: 70%;
+    margin-top: 5%;
+    padding: 24px;
+    padding-bottom: 5rem;
+    border-radius: 16px;
+    background: rgb(255, 255, 255);
+    -ms-overflow-style: none; /* for Internet Explorer, Edge */
+    scrollbar-width: none; /* for Firefox */
+    overflow-y: scroll;
+  }
+
+  .pops-project::-webkit-scrollbar {
+    display: none; /* for Chrome, Safari, and Opera */
+  }
+
+  .pops-project img {
+    margin-top: 0.5rem;
+  }
+
+  .pops-project .text {
+    margin-bottom: -0.4rem;
+    width: 70%;
+  }
+
+  .pops-project .languages {
+    width: 22%;
+    position: absolute;
+    top: 81%;
+    left: 70%;
+    border-bottom: 1px solid #ebecf0;
+    padding-bottom: 1rem;
+  }
+
+  .pops-project .btns {
+    position: absolute;
+    top: 91%;
+    left: 69%;
+    width: 25%;
+    display: flex;
+    justify-content: space-around;
+  }
+
+  #pop-xmark {
+    position: absolute;
+    left: 81.9%;
+    top: 11%;
+    font-size: 24px;
+    color: #67798e;
+  }
+
+  #tempate-image {
+    width: 100%;
+    height: 386px;
+  }
+
+  #caption {
+    display: flex;
+    align-items: center;
+  }
+
+  .btns img {
+    margin-left: 0.5rem;
+    margin-bottom: 0.5rem;
+  }
+
+  #caption img {
+    margin-bottom: 0.7rem;
+  }
+
+  .btns i {
+    font-size: 1rem;
+    margin-left: 0.3rem;
+  }
+
+  .btns button {
+    display: flex;
+    width: 5rem;
+    padding: 0;
+    align-items: center;
+    justify-content: center;
+    font-size: 0.5rem;
+  }
```

Comment:
- [ ] [OPTIONAL] Great job 👍 . Kindly adjust the `font-size` of your text as it is hard to see and would be much harder for people with eye problems 🙏 

![image](https://user-images.githubusercontent.com/74814780/180257648-52bf7c43-fba4-4d60-a48c-dc8978256a4c.png)